src/utils/os/index.ts

```diff
@@ -1,10 +1,19 @@
-import { exec, ExecOptions } from 'child_process';
+import { exec, execSync, ExecOptions } from 'child_process';
 import { Readable } from 'stream';
-import logger from '../../logger';
+import logger from '../../logger/index.js';
+
+export function runCommandSync (command: string, opts?: ExecOptions) {
```

Comment:
We might want to live stream back stdout so the user doesn't think the process has stalled.  Checkout how we did that in precloud: https://github.com/tinystacks/precloud/blob/main/src/utils/os/index.ts